### PR TITLE
Create topic for PayPal Express Checkout payment method

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) creates a topic devoted to the PayPal Express Checkout payment method. Previously, there was a PayPal topic that described multiple PayPal payment methods that did not have the same workflow as Express Checkout.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/payment-methods/paypal-express-checkout.html

whatsnew
Added the [PayPal Express Checkout payment method](https://devdocs.magento.com/guides/v2.3/graphql/payment-methods/paypal-express-checkout.html)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=5099) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=5099) before we can accept your contribution.<br/>**3** out of **4** committers have signed the CLA.<br/><br/>:white_check_mark: keharper<br/>:white_check_mark: jeff-matthews<br/>:white_check_mark: sdzhepa<br/>:x: Lori Krell<hr/>**Lori Krell** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: guides/v2.3/graphql/mutations/create-paypal-express-token.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+### PaypalExpressTokenInput {#PaypalExpressTokenInput}
+
+The `PaypalExpressTokenInput` object defines the attributes required to receive a payment token from PayPal.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`cart_id` | String! | The unique ID that identifies the customer's cart
+`code` | String! | Payment method code
+`express_button` | Boolean | Indicates whether the buyer selected the PayPal Express Checkout button. The default value is `false`
+`urls` | [`PaypalExpressUrlsInput!`](#PaypalExpressUrlsInput)! | A set of relative URLs that PayPal will use in response to various actions during the authorization process.
```
Comment (erikmarr) [inline guides/v2.3/graphql/mutations/create-paypal-express-token.md]:
```suggestion
`urls` | [`PaypalExpressUrlsInput!`](#PaypalExpressUrlsInput) | A set of relative URLs that PayPal will use in response to various actions during the authorization process.
```

File: guides/v2.3/graphql/mutations/create-paypal-express-token.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+`use_paypal_credit` | Boolean | Indicates whether the buyer clicked the Paypal credit button. The default value is `false`
```
Comment (erikmarr) [inline guides/v2.3/graphql/mutations/create-paypal-express-token.md]:
```suggestion
`use_paypal_credit` | Boolean | Indicates whether the buyer clicked the PayPal credit button. The default value is `false`
```

File: guides/v2.3/graphql/payment-methods/paypal-express-checkout.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+![PayPal Express Checkout sequence diagram]({{site.baseurl}}/common/images/graphql/paypal-express-checkout.svg)
+
+The following steps describe the flow of calls required to complete a typical PayPal Express Checkout authorization. A successful purchase requires that you send three mutations to PayPal, and the buyer must approve the purchase by logging in to PayPal.
+
+{% include graphql/payment-methods/paypal-express-checkout-workflow.md %}
+
+## Additional Payment information
+
+When you set the payment method code to `paypal_express` in the `setPaymentMethodOnCart` mutation, you must also specify attributes specific to this payment method in `additional_data` object. These attributes are defined in the   `paypal_express` object:
```
Comment (erikmarr) [inline guides/v2.3/graphql/payment-methods/paypal-express-checkout.md]:
```suggestion
When you set the payment method code to `paypal_express` in the `setPaymentMethodOnCart` mutation, you must also specify attributes specific to this payment method in `additional_data` object. These attributes are defined in the  `paypal_express` object:
```

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
running tests

**Pull request merged**